src/System.Composition.Convention/tests/System.Composition.Convention.Tests.csproj

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>  
+    <KeepAllProjectReferences>true</KeepAllProjectReferences>  
```

Comment:
@Chrisboh @MattGal for test projects that don't get converted to using package references do we intend for them to still be included as part of one of these test runs or do we expect to filter them out?